### PR TITLE
[Tizen] Update Tizen SDK to 8.0

### PR DESCRIPTION
**Pull request by jlatusek**
## Problem

CI images are using the old version of Tizen 7.0

## Solution

Update tizen to latest avaliable version - 8.0

### CODE REVIEW

File: integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh [hunk 1]
```diff
@@ -232,13 +232,14 @@ function install_tizen_sdk() {
...
-        'libnsd-dns-sd-*.armv7l.rpm')
+        'libnsd-dns-sd-*.armv7l.rpm'
+        'sensord-*.armv7l.rpm')
     download "$URL" "${PKG_ARR[@]}"
 
     # Tizen Developer Platform Certificate
     URL="http://download.tizen.org/sdk/extensions/Tizen_IoT_Headless/binary/"
     PKG_ARR=(
-        "$TIZEN_VERSION-iot-things-add-ons_*_ubuntu-64.zip")
+        "7.0-iot-things-add-ons_*_ubuntu-64.zip")
```
Comment (arkq) [inline integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh:242]: Please add comment why this is not updated

Comment (jlatusek) [reply integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh:243]: Fixed

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by andy31415** [comment]
@jlatusek is this supposed to still stay in draft or is it ready to review?

A comment in the summary of why something is in draft (until when) would be very helpful.

**Comment by arkq** [comment]
It still waits for my confirmation that after the upgrade everything works correctly. Unfortunately I have some troubles with upgrading my device to Tizen 8.0. I hope I will verify that on Monday.

**Review by arkq — APPROVED** [review]
Everything works as expected with Tizen 8.0

**Pull request merged**